docs/articles/nunit/writing-tests/attributes/testcasesource.md

```diff
@@ -17,10 +17,12 @@ Consider a test of the divide operation, taking three arguments: the numerator,
 
 ### Form 1 - [TestCaseSource(string sourceName)]
 
+#### Note: We use [nameof(DivideCases)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/nameof) here to avoid introducing [magic strings](https://en.wikipedia.org/wiki/Magic_string) into our code, which offers better resilience when refactoring.
```

Comment:
```suggestion
> [!NOTE]
> We use [the `nameof` operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/nameof) to avoid introducing [magic strings](https://en.wikipedia.org/wiki/Magic_string) into code, which offers better resilience when refactoring. While `nameof` is recommended, you could also use the string "DivideCases" to achieve the same outcome.
```

Suggesting:

* Updated format (we have a concept for notes that works within docfx)
* Minor language changes
* Indicating what the string-based approach would be.

Reply:
I like this suggestion! I can't make another suggestion on top of it, but could we also remove the `/en-us/` section of the docs.microsoft.com link? That means that users who normally use a translated version of docs.ms will be directed to the page in their default language, rather than specifically the en-us version. 🙂 

Thanks for doing this @svengeance - I think it's a great change overall. 😄

Reply:
Great catch @ChrisMaddock -- I updated my suggestion so @svengeance can merge it in if they choose.

Reply:
Thanks Sean!